### ISSUE TITLE
Samplesheets/Landingzones: Support JSON data from SODAR Taskflow

### ISSUE THREAD

**Issue by mikkonie** [bihealth/sodar-server#1233]
Related to sodar_taskflow#83.

**Comment by mikkonie** [comment]
Closing. Would require too much work and changes to SODAR Core for authentication, instead will postpone dependent issues until after #691. See sodar_taskflow#84 for more info.
